saleor/graphql/account/mutations/staff/customer_create.py

```diff
@@ -38,3 +52,70 @@ class Meta:
                 description="Setting a new password for the account is requested.",
             ),
         ]
+
+    @classmethod
+    @traced_atomic_transaction()
+    def save(cls, info: ResolveInfo, instance, cleaned_input):
+        manager = get_plugin_manager_promise(info.context).get()
+
+        default_addresses = cls.save_and_apply_addresses_from_input(
+            cleaned_input=cleaned_input, user_instance=instance
+        )
+
+        instance.save()
```

Comment:
Do we need tis save at all? It seems like save from line 74 should be enough

Reply:
It fails without both of them, because (according to my understanding) some objects must be saved before m2m are saved